src/expander/expander_utils2.c

```diff
@@ -18,7 +18,7 @@ t_bool	is_print_dollar(char *s, int i, t_quote q_stt)
 	{
 		if (IN_QUOTE == q_stt.double_quote && DOUBLE_QUOTE_CHAR == s[1])
 			return (TRUE);
-		if (!s[1] || expander_is_space(s[1]))
+		if (!s[1] || expander_is_space(s[1]) || SLASH_CHAR == s[1])
```

Comment:
Myabe directly
`if (!s[1] || is_end_of_var_name(s[1]))`

Reply:
the change is about printing a stand alove $ in this case: `echo $/`

before change
```
msh$ echo $HOME/hey
/home/yoav/hey
msh$ echo $/
/
msh$ 
```

after change
```
msh$ echo $HOME/hey
/home/yoav/hey
msh$ echo $/
$/
msh$ 
```